app/src/main/java/com/example/beerapp/API/RetrofitConnector.java

```diff
@@ -0,0 +1,25 @@
+package com.example.beerapp.API;
+
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class RetrofitConnector {
```

Comment:
this class should be a singleton. There is no need to create retrofit per each service. 
If you would like to use kotlin it will be an object class or if you add Koin to inject depenecies it will be implemented as singleton